doc/Core.xml

```diff
@@ -1270,9 +1286,13 @@
       </section>
       <section>
         <title>Firmware Upgrade</title>
-        <para>Two mechanisms are provided for upgrading the firmware on a device. The first uses the UpgradeSystemFirmware command to send the new firmware image using MTOM.</para>
+        <para>Three mechanisms are provided for upgrading the firmware on a device. The first uses
+          the UpgradeSystemFirmware command to send the new firmware image using MTOM. The third one
+          enables firmware upgrades for cloud connected devices.</para>
         <para>The second is a two stage process; first the client sends the StartFirmwareUpgrade command to instruct the device to prepare for upgrade, then it sends the firmware image using HTTP POST.</para>
         <para>The HTTP method is designed for resource-limited devices that may not be capable of receiving a new firmware image in its normal operating state.</para>
+        <para>The third method involves an interaction between OCS, MCS and the device, in order to
+          trigger the firmware upgrade.</para>
```

Comment:
Great that you place it here, but it is very hard to understand. Please elaborate a bit more and avoid unfamiliar acronyms.

Reply:
Agreed. Here MCS and OCS should be expanded to their full meaning and also added as updated to the Definitions with link / reference to the Cloud Integration Spec. If acronyms are needed then MCS & OCS should be added to the Abbreviations.

Reply:
See my commit